ifrs17/Report/ParameterReportMutableScopeInteractive.ipynb

```diff
@@ -0,0 +1,246 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "XdWtZAfDUkKgJxSlrhCSSw",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"ReportMutableScopesInteractive\"",
+                "\n#!import \"ParameterReportsQueries\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public interface RepFormsEntity : MutableScopeWithWorkspace",
+                "\n{",
+                "\n    [DropdownMethod(nameof(ReportMenu))]",
+                "\n    string ReportType{get; set;}",
+                "\n",
+                "\n    IReadOnlyCollection<string> ReportMenu() => new string[] {\"\", \"DataNodeReport\", ",
```

Comment:
Introduce some consts instead of hardcoding

Reply:
Rendered this an enumerable. I believe it is OK

Reply:
Move it to the constant class and do the same exactly with the import types and forms.

Reply:
OK, now it is all rearranged in the classes, so I can close the issue.